doc/release-notes-11413.md

```diff
@@ -0,0 +1,11 @@
+Updated or changed RPC
+----------------------
+
+The `bumpfee`, `fundrawtransaction`, `sendmany`, `sendtoaddress`, and `walletcreatefundedpsbt`
+RPC commands have been updated to include two new fee estimation methods "BTC/kB" and "sat/B".
```

Comment:
5ee0e46 nit: missing comma after "methods"

Reply:
I think it's okay as it is. The methods `a` and `b` were added.